x-pack/legacy/plugins/apm/public/components/shared/charts/TransactionCharts/TransactionLineChart/index.tsx

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useMemo } from 'react';
+import { flatten } from 'lodash';
+import d3 from 'd3';
+import {
+  Coordinate,
+  RectCoordinate
+} from '../../../../../../typings/timeseries';
+import { useChartsTime } from '../../../../../hooks/useChartsTime';
+import { getEmptySeries } from '../../CustomPlot/getEmptySeries';
+// @ts-ignore
+import CustomPlot from '../../CustomPlot';
+import { toQuery, fromQuery } from '../../../Links/url_helpers';
+import { history } from '../../../../../utils/history';
+
+interface Props {
+  series: Array<{
+    color: string;
+    title: React.ReactNode;
+    titleShort?: React.ReactNode;
+    data: Array<Coordinate | RectCoordinate>;
+    type: string;
```

Comment:
Isn't this a duplicate of https://github.com/elastic/kibana/blob/1e1153ea7b11b543f9399267b8e3173883258ced/x-pack/legacy/plugins/apm/typings/timeseries.ts#L17-L27 ?

Reply:
Or rather: can they be combined?